src/main/java/net/imagej/ops/features/hog/HistogramOfOrientedGradients2D.java

```diff
@@ -124,33 +144,73 @@ public void convert(final T arg0, final FloatType arg1) {
 
 	@Override
 	public boolean conforms() {
-		return in().numDimensions() == 2;
+		return ((in().numDimensions() == 2) || ((in().numDimensions() == 3) && (in().dimension(2) == 3)));
```

Comment:
is this really a requirement that `in().dimension(2)==3`? actually it should work for arbitrary dims, right?